source/NukeBuildComponents/IOctopusNukeBuild.cs

```diff
@@ -13,9 +12,9 @@ public interface IOctopusNukeBuild : INukeBuild
 
         Enumeration Config { get; }
 
-        [Solution] Solution Solution => ValueInjectionUtility.TryGetValue(() => Solution);
+        [Solution] Solution? Solution => TryGetValue(() => Solution);
 
-        [OctoVersion] OctoVersionInfo OctoVersionInfo => ValueInjectionUtility.TryGetValue(() => OctoVersionInfo);
+        [OctoVersion] OctoVersionInfo? OctoVersionInfo => TryGetValue(() => OctoVersionInfo);
```

Comment:
I think that making this nullable might be causing `IPackComponent` to set empty string outputs for `semver` and `prerelease_tag`, which is causing GHA to fail on the tag action (and that seems to have pushed a `v1.0.0` for some reason...)

Reply:
Yeah, see [here](https://github.com/OctopusDeploy/NukeBuildComponents/runs/5843967911?check_suite_focus=true#step:4:139) - `null` in the output.